usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -431,6 +433,7 @@ def post(self, request, format=None):
         sort_direction = request_data['sort']['direction']
         fy = request_data['filters']['fy']
         keyword = request_data.get('keyword', None)
+        agency_id = request_data.get('agency_identifier', None)
```

Comment:
This is a broader change, but it would probably make more sense to make 'agency_identifier' a sub parameter of 'filter', i.e., the way 'fy' is currently handled.  Having it as a top level parameter makes the request a little bit muddled. If you have questions about how this would work, I can help.